locator/util.cc

```diff
@@ -148,4 +147,11 @@ describe_ring(const replica::database& db, const gms::gossiper& gossiper, const
     co_return ranges;
 }
 
+future<std::vector<dht::token_range_endpoints>>
+describe_ring(const replica::database& db, const gms::gossiper& gossiper, const sstring& keyspace, bool include_only_local_dc) {
+    //Token.TokenFactory tf = getPartitioner().getTokenFactory();
```

Comment:
I think we can drop this comment. It probably remained here from the time when we rewrote C* Java code to C++.

Reply:
added a dedicated commit to drop this comment.